### DISCUSSION TITLE
Managed Dependencies

### DISCUSSION THREAD

**Discussion by ralmeida-espatial** [ben-manes/caffeine#867] [Q&A] [answered]
[caffeine 3.1.2](https://mvnrepository.com/artifact/com.github.ben-manes.caffeine/caffeine/3.1.2) has a lot of Managed Dependencies. Is this an error?! I would think so... In a project I use an old version of protobuf (because of other dependencies) and it's getting version conflicts using this caffeine version. Versions [3.1.1](https://mvnrepository.com/artifact/com.github.ben-manes.caffeine/caffeine/3.1.1) and bellow don't have this problem...

**Comment by ben-manes** [comment, marked as answer]
Yes, it was an accidental metadata leak which should be resolved in https://github.com/ben-manes/caffeine/commit/396a9405f52d8f7bd3baa162fb6f1a9f5a239c91 and https://github.com/ben-manes/caffeine/commit/7321a56a0fe92308c5b20c2a789ab77907ca480e. The security scanners were warning about test libraries and build plugin transitive dependencies, I suppose because exploits of CI/CD are becoming more common threats now. The dependency constraints leaked into the external metadata. I was hoping to do a final review of the changes and release this week.

**Reply by ben-manes** [reply]
Released 3.1.3

**Reply by ralmeida-espatial** [reply]
Thank you. BTW, do you know how long it usually takes to show up in maven repository?

**Reply by ben-manes** [reply]
It usually takes about 10 minutes to get into the repository and an hour to get indexed for the search ui. Sometimes search takes longer or I’ll have to file a ticket, but getting into the repository is always within 1-2 hours at worst.

**Reply by ben-manes** [reply]
Oh I meant central (maven.org), I don’t know about mvnrepository which is unrelated and often confused as if official. Probably a day or two.
